src/data/articles.yaml

```diff
@@ -0,0 +1,19 @@
+- title: Wiki info about the city of Portland, OR
+  author: Author Name
+  date: May 5th, 2016
```

Comment:
I know we're not doing this consistently at the moment, but it might be beneficial to have all dates use the `yyyy-mm-dd` format (or at least use a format that works with `new Date()`).